phpstan.neon

```diff
@@ -11,10 +11,7 @@ parameters:
 		- src/
 		- plugin.php
 
-#    ignoreErrors:
-#        - '#PHPDoc tag @var#'
-#
-#    excludePaths:
-#        - ./*/*/FileToBeExcluded.php
-#
-#    checkMissingIterableValueType: false
+	scanDirectories:
+		- %currentWorkingDirectory%/../wordpress-fieldmanager/
+		- %currentWorkingDirectory%/vendor/campaignmonitor/
```

Comment:
Yep! To prevent "unknown class" phpstan errors for `CS_REST_General` and `CS_REST_Wrapper_Result`

Reply:
I'm a bit over my head with this but I have a feeling that maybe adding vendor folders may not be the best way to go for this? Wondering if we can configure phpstan to ignore those specifically?

Use PHPStan's ignoreErrors configuration: If you want to suppress specific "unknown class" errors without modifying the scanDirectories or scanFiles, you can use the ignoreErrors configuration option. This allows you to specify patterns or specific errors to ignore. Example:

```yaml
parameters:
    ignoreErrors:
        - '#Unknown class MyVendor\\MyPackage\\MyClass#'
```
Adding vendor files to PHPStan's `scanDirectories` _seems_ like it might negatively impact performance over time, and seems like it should be unnecessary given proper dependency management, but I am new to phpstan, so maybe asking another dev would be good here.

Reply:
Okay I'll ask in #opensource – I was looking at other projects `phpstan.neon` files and found a bunch of them scanning other plugin files too: https://github.com/alleyinteractive/wpr/blob/production/phpstan.neon

Reply:
@srtfisher Do you want to offer insight here?